### PR TITLE
quick follow-up fixes

### PR DESCRIPTION
**Pull request by mchoi8739**
*Issue #, if available:*

*Description of changes:*

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

- [ ] I have read the [CONTRIBUTING](https://github.com/aws/sagemaker-jumpstart-industry-pack/blob/main/CONTRIBUTING.md) doc
- [ ] I used the commit message format described in [CONTRIBUTING](https://github.com/aws/sagemaker-jumpstart-industry-pack/blob/main/CONTRIBUTING.md#committing-your-change)
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have updated any necessary [documentation](https://github.com/aws/sagemaker-jumpstart-industry-pack/blob/main/README.rst) (if appropriate)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by mchoi8739** [comment]
staged preview: https://sagemaker-jumpstart-industry-pack--24.org.readthedocs.build/en/24/index.html

**Review by johnhe-dev — APPROVED** [review]

File: docs/source/what-is.rst [hunk 1]
```diff
@@ -22,11 +22,25 @@ in the ML lifecycle.
    :alt: An architecture diagram of SageMaker JumpStart end-to-end solutions
        and the coverage of SageMaker JumpStart Industry Python SDK
 
-#. Use SageMaker JumpStart Industry notebooks for solutions, models, and examples.
+#. Use `SageMaker JumpStart Industry
+   <https://docs.aws.amazon.com/sagemaker/latest/dg/studio-jumpstart-industry.html>`_
+   notebooks for solutions, models, and examples.
```
Comment (sophiayue1116) [inline docs/source/what-is.rst:27]: the notebooks in the repo is not used in the solution

Comment (mchoi8739) [reply docs/source/what-is.rst:27]: Yes, I understand that. Here, I generally spoke about the entire smjsindustry notebooks.

Comment (mchoi8739) [reply docs/source/what-is.rst:27]: I am not talking that the notebook in the repo is what's in the solution. I am gererally saying that the solutions, models, and example notebooks are available to use.

Comment (mchoi8739) [reply docs/source/what-is.rst:27]: https://sagemaker-jumpstart-industry-pack--24.org.readthedocs.build/en/24/what-is.html#how-it-works

Screen Shot 2021-09-29 at 5 30 34 PM

Comment (mchoi8739) [reply docs/source/what-is.rst]: @sophiayue1116 Rephrased it: https://sagemaker-jumpstart-industry-pack--24.org.readthedocs.build/en/24/what-is.html#how-it-works

**Review by sophiayue1116 — APPROVED** [review]

**Pull request merged**